src/collective/sitter/locales/collective.sitter.pot

```diff
@@ -188,12 +244,22 @@ msgstr ""
 msgid "desc_gender"
 msgstr ""
 
+#. Default: "Ordner Eigenschaften bearbeiten"
+#: collective.sitter/src/collective/sitter/browser/templates/sittermanager.pt
+msgid "edit_sitter_folder"
+msgstr ""
+
 #. Default: "Erfahrung"
 #: collective.sitter/src/collective/sitter/browser/templates/sitterview.pt
 #: collective.sitter/src/collective/sitter/content/sitterfolder.py
 msgid "experiences"
 msgstr ""
 
+#. Default: "FAQ"
+#: collective.sitter/src/collective/sitter/browser/templates/sittermanager.pt
+msgid "fqa"
```

Comment:
Es heißt immernoch an mehreren Stellen "fqa".

Reply:
all fqas are gone
no plan why this always happpens. 
I nearly added a few new ones during translation